src/core/border_router/routing_manager.cpp

```diff
@@ -602,16 +602,11 @@ void RoutingManager::SendRouterAdvertisement(RouterAdvTxMode aRaTxMode)
     }
 
     mRxRaTracker.SetHeaderFlagsOn(header);
+    header.SetSnacRouterFlag();
```

Comment:
If the Thread BR also functions as a CE router (per RFC 7084) it will have its own rules for network configuration that differ from the SNAC router. In this case it must advertise S=0.  So a build switch to configure the build for a CE router could be useful - in this case S=0 is set.   Agree here that the default must be S=1.

In the future someone may want to make a device that can dynamically switch between CE router mode and SNAC router mode, e.g. depending on whether the WAN is plugged in. But that could be done with a new/future PR I think if anyone would need that.

Reply:
> In other words, a "platform" that handles RAs while the Thread side code is prevented from emitting RAs.

> But that could be done with a new/future PR if anyone needs that functionality.

Yes, exactly. We can leave this for new/future PRs when/if there's a need for such use-cases. We could add an `OPENTHREAD_CONFIG` option to control this or perhaps an API.